door-open-service/esp8266-nodemcu/init.lua

```diff
@@ -13,16 +13,18 @@ gpio.write(gpio_pin, gpio.LOW)
 
 -- 3. Set-up timer
 function setup_alarm()
```

Comment:
I changed it to `setup_wifi`. It's still not very good as it also connects to the backend, but I think it's good enough :P